scss/_accordion.scss

```diff
@@ -20,10 +21,10 @@
   &:not(.collapsed) {
     color: $accordion-button-active-color;
     background-color: $accordion-button-active-bg;
-    box-shadow: inset 0 ($accordion-border-width * -1) 0 $accordion-border-color;
+    // box-shadow: inset 0 ($accordion-border-width * -1) 0 $accordion-border-color;
```

Comment:
I think this is for a11y. Might want to keep it or show enough visual contrast some other way to indicate section is active.